src/main/java/io/kokuwa/maven/helm/StripSensitiveDataLog.java

```diff
@@ -0,0 +1,118 @@
+package io.kokuwa.maven.helm;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.maven.plugin.logging.Log;
+
+/**
+ * Delegate for maven {@link Log} to strip sensitive data before logging.
+ *
+ * @author Stephan Schnabel
+ */
+public class StripSensitiveDataLog implements Log {
+
+	private static final Pattern HELM_PASSWORD = Pattern.compile("^.*--password=(?<secret>[^-\\s]+).*$");
+	private static final Pattern KUBE_TOKEN = Pattern.compile("^.*--kube-token (?<secret>[^-\\s]+).*$");
+	private static final List<Pattern> PATTERNS = Arrays.asList(HELM_PASSWORD, KUBE_TOKEN);
+	private static final String REPLACEMENT = "*****";
+
+	static String strip(CharSequence content) {
+		String stripped = content.toString();
+		for (Pattern pattern : PATTERNS) {
+			Matcher matcher = pattern.matcher(content);
+			if (matcher.matches()) {
+				stripped = stripped.replace(matcher.group("secret"), REPLACEMENT);
+			}
+		}
+		return stripped;
+	}
+
+	private final Log delegate;
+
+	public StripSensitiveDataLog(Log delegate) {
+		this.delegate = delegate;
+	}
+
+	@Override
+	public void debug(CharSequence content) {
+		delegate.debug(strip(content));
+	}
+
+	@Override
+	public void debug(CharSequence content, Throwable error) {
+		delegate.debug(content, error);
```

Comment:
why missing `strip` here and in other methods?

Reply:
This is a mistake, see #167.